mu/editor.py

```diff
@@ -189,6 +189,11 @@ def save(self):
             tab.path = self._view.get_save_path(MICROPYTHON_DIRECTORY)
         if tab.path:
             # The user specified a path to a file.
+
+            if '.' not in tab.path.split('/')[-1]:
```

Comment:
Could you do this instead?

```
if not os.path.basename(tab.path).endswith('.py'):
```

Why? Because we'd be using built-in functions designed for this sort of check rather than rolling our own platform specific check (what you have won't work on Windows ~ it uses the path separator `\`).

Will merge once fixed. Great stuff! :smile: